From f714d2925caed58936d6e068ca206cf40499af37 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sat, 11 Mar 2017 20:34:22 -0500
Subject: [PATCH 03/14] Drop btrfs-convert (Closes: #824895, #854489)

Rationale: the test failed
...
    [TEST]   convert-tests.sh
    [TEST/conv]   001-ext2-basic
    [TEST/conv]     ext2 4k nodesize, btrfs defaults
failed: /scratch/btrfs-progs-v4.10/btrfs-convert --rollback /scratch/btrfs-progs-v4.10/tests/test.img
test failed for case 001-ext2-basic
Makefile:271: recipe for target 'test-convert' failed
make: *** [test-convert] Error 1
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 6be097c8..f6181164 100755
--- a/debian/rules
+++ b/debian/rules
@@ -19,7 +19,7 @@ override_dh_autoreconf:
 	dh_autoreconf ./autogen.sh
 
 override_dh_auto_configure:
-	dh_auto_configure -- --bindir=/bin
+	dh_auto_configure -- --bindir=/bin --disable-convert
 
 override_dh_auto_build:
 	dh_auto_build -- V=1
-- 
2.11.0

